systemd/ceph-exporter.service.in

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Ceph cluster exporter daemon
+PartOf=ceph.target
+After=network-online.target local-fs.target
+Before=ceph.target
+Wants=network-online.target local-fs.target ceph.target ceph-mon.target
+
+[Service]
```

Comment:
Only one instance of ceph exporter must run in the host. This means that probably it will be required to add the following service directives:
[Service]
Type=simple
ExecStartPre=/bin/bash -c '! pgrep -x "ceph-exporter"'

Reply:
@jmolmo  Systemd serves as service manager here and ensures there is only one instance running (via the systemd service unit). See example #2 at https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Examples.

Since systemd "knows" which processes is started via the units (see https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Type=), it also knows which ones to stop or restart.

I don't think service units should be expecting other, indepently launched  "rogue"  `ceph-exporter` processes to exist..

Reply:
Ideally only 1 instance should be running at a time else there'd be duplicates of performance metrics exposed, as any instance would be basically accessing same socket file path (on that node/host). So I'd advice to have only 1 instance per node.

Reply:
@jmolmo I agree that only one ceph-exporter instance should be running on a single node.
However, my understanding is the same as @frittentheke 's comment, so I think we don't need to worry about the multiple instances being launched by systemd.

Reply:
@frittentheke thanks for the explanation! (I didn't know  type=simple is used by default). Then I think everything on place. :-)